Sources/Configuration.swift

```diff
@@ -8,7 +8,10 @@
 
 import Foundation
 
-let DEFAULT_SERVER = "cdn.contentful.com"
+enum Defaults {
```

Comment:
I had to google that one! swiftlint through an error on having an `_` in a variable